src/components/modals/save-form.component.tsx

```diff
@@ -120,12 +121,27 @@ const SaveForm: React.FC<SaveFormModalProps> = ({ form, schema }) => {
       }
     } else {
       try {
-        if (form?.resources.length != 0) {
-          deleteClobdata(form?.resources[0].valueReference);
-          deleteResource(form?.uuid, form?.resources[0].uuid);
+        if (form?.resources?.length != 0) {
+          const valueReference = form?.resources[0].valueReference ?? "";
+
+          deleteClobdata(valueReference)
+            .catch((error) =>
+              console.error("Unable to delete clobdata: ", error)
```

Comment:
I think it would be best to alert the user that an error has occurred, by either showing the `Toast` or `Notification`

Reply:
Sure